src/openforms/registrations/contrib/objects_api/tests/test_backend_v2.py

```diff
@@ -168,6 +171,157 @@ def test_submission_with_objects_api_v2(self):
             },
         )
 
+    def test_submission_with_objects_api_v2_with_payment_attributes(self):
+        submission = SubmissionFactory.from_components(
+            [
+                # fmt: off
+                {
+                    "key": "age",
+                    "type": "number"
+                },
+                {
+                    "key": "lastname",
+                    "type": "textfield",
+                },
+                {
+                    "key": "location",
+                    "type": "map",
+                },
+                # fmt: on
+            ],
+            completed=True,
+            submitted_data={
+                "age": 20,
+                "lastname": "My last name",
+                "location": [52.36673378967122, 4.893164274470299],
+            },
+        )
+
+        v2_options: RegistrationOptionsV2 = {
+            "version": 2,
+            "objects_api_group": self.objects_api_group,
+            # See the docker compose fixtures for more info on these values:
+            "objecttype": UUID("8e46e0a5-b1b4-449b-b9e9-fa3cea655f48"),
+            "objecttype_version": 3,
+            "upload_submission_csv": True,
```

Comment:
disabling this and all the objecttypes cuts down massively on API calls that are not relevant to this test anyway, so please try to set up your tests for only the minimal aspect that's being tested